nautobot/core/testing/mixins.py

```diff
@@ -155,6 +155,18 @@ def add_permissions(self, *names):
             obj_perm.users.add(self.user)
             obj_perm.object_types.add(ct)
 
+    def add_permission(self, name, **kwargs):
```

Comment:
Maybe just add `**kwargs` support to `add_permissions()` above? This seems functionally equivalent for the case where there's only a single `name`.